_specification-v1/manifest.md

```diff
@@ -52,9 +30,9 @@ nav_order: 25
 </table>
 
 Defines the type of Origami project that the manifest belongs to. **Must** be set to one of:
-- `"component"` or `"module"`: A front-end component that follows [the component specification](/spec/v1/components/)
+- `"component"` or `"module"`: A front-end component that follows [the component specification](/spec/v2/components/)
```

Comment:
```suggestion
- `"component"`: A front-end component that follows [the component specification](/spec/v2/components/)
```
Now would be a good time to drop "module" for component, which also matches the query param etc of the Build Service. 

We would need to update obt:
- https://github.com/Financial-Times/origami-build-tools/pull/935#discussion_r617384013

And repo data, we could normalise to "module" for v1 components and "component" for v2 components to avoid breaking changes:
https://github.com/Financial-Times/origami-repo-data/blob/af5176a8fb58a7522ea68e976e7be222a8600adf/lib/create-version-from-ingestion/normalise-origami-manifest.js#L20
 
Projects that rely on repo data would need to be updated to handle v2 components, i.e. the registry.

...and probably other places.

Reply:
Switching from "module" to "component" isn't part of the migration step so [some have](https://github.com/Financial-Times/o-colors/pull/300) been updated and [some haven't](https://github.com/Financial-Times/o-autoinit/pull/130/files#diff-8246ac3cb6728b57fe6239256a401fe4ee55bc84aedc47cd644b215d10eac898R2).

Reply:
I'm happy to drop `module` as it doesn't effect the migration steps for Origami users

Reply:
i want to use `component`